go/base-arm/Dockerfile.tmpl

```diff
@@ -59,7 +59,8 @@ RUN \
 
 ARG GOLANG_VERSION=1.24.1
 {{- if eq .FIPS "true"}}
-ARG GOLANG_DOWNLOAD_URL=https://aka.ms/golang/release/latest/go$GOLANG_VERSION-1.linux-arm64.tar.gz
+ARG SECURITY_VERSION=-1
```

Comment:
Decided to use the security_version arg to help with the automation, using the same mechanism we have done for the GOLANG_VERSION arg